src/hotspot/cpu/arm/arm.ad

```diff
@@ -804,16 +804,16 @@ uint MachSpillCopyNode::implementation( CodeBuffer *cbuf,
 
 #ifndef PRODUCT
 void MachSpillCopyNode::format( PhaseRegAlloc *ra_, outputStream *st ) const {
-  implementation( NULL, ra_, false, st );
+  implementation( nullptr, ra_, false, st );
```

Comment:
It's the square with the +- in it to the right of "Preview".

Reply:
The only trick with this button is that if you commit the suggestion, you have to pull it back to your local repo in case you want to make more changes there.

Also if you use the suggestion button, you become co-author, which is a bit odd.